arcgis-ios-sdk-samples/Layers/Apply unique values with alternate symbols/README.md

```diff
@@ -0,0 +1,46 @@
+# Apply unique values with alternate symbols
+
+Apply a unique value with alternate symbols at different scales.
+
+![Apply unique values with alternate symbols](apply-unique-values-alternate-symbols.png)
+
+## Use case
+
+When a layer is symbolized with unique value symbology, you can specify the visible scale range for each unique value. This is an effective strategy to limit the amount of detailed data at smaller scales without having to make multiple versions of the layer, each with a unique definition query.
+
+Once scale ranges are applied to unique values, you can further refine the appearance of features within those scale ranges by establishing alternate symbols to different parts of the symbol class scale range.
+
+## How to use the sample
+
+Zoom in and out of the map to see alternate symbols at each scale. The symbology changes according to the following scale ranges: 0-5000, 5000-10000, 10000-20000. To go back to the initial viewpoint, tap "Reset viewpoint".
```

Comment:
```suggestion
Zoom in and out of the map to see alternate symbols at each scale. The symbology changes according to the following scale ranges: 0-5,000, 5,000-10,000, 10,000-20,000. To go back to the initial viewpoint, tap "Reset viewpoint".
```

Also I wonder if surrounding spaces need to be added to the hyphen when it indicates a range in English writing.

Reply:
I don't think there's a hard rule about that. It's stylistic and personal preference